pkg/integration/utils/cli.go

```diff
@@ -0,0 +1,82 @@
+//go:build integration
+// +build integration
+
+package utils
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"testing"
+)
+
+// use the CLI and call the command `docker pull` for each image in a list of images
```

Comment:
Do we want to add reasoning why we're not using the docker library? I'm not saying that we should. Just thought it could be nice to have it mentioned.

Reply:
Mostly for simplicity, I updated the comment to clarify that the fact we use CLI is secondary.